ethcore/Cargo.toml

```diff
@@ -16,9 +16,9 @@ crossbeam = "0.3"
 ethash = { path = "../ethash" }
 ethcore-bloom-journal = { path = "../util/bloom" }
 parity-bytes = "0.1"
-hashdb = "0.2.1"
-memorydb = "0.2.1"
-patricia-trie = "0.2"
+hashdb  = "0.3.0"
```

Comment:
why double space before `=` here and below?

Reply:
Because I can't grep so well. Sigh. Fixing.